src/DataBaseHelper.java

```diff
@@ -0,0 +1,38 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/* Here I am storing all the user given data and results along with data and time of the operation into database*/
+class DataBaseHelper {
+
+    protected DataBaseHelper(String word, String filepath, String resultToDatabase, int totalNoOfWords, String errorMessage) throws SQLException {
+        storingDataToDataBase(word, filepath, resultToDatabase, totalNoOfWords, errorMessage);
+    }
+
+    private static void storingDataToDataBase(String word, String filepath, String resultToDatabase, int totalNoOfWords, String errorMessage) throws SQLException {
+        Connection connectionToDataBase = null;
+        String driveClass = "com.mysql.cj.jdbc.Driver";
```

Comment:
```suggestion
        String driverClass = "com.mysql.cj.jdbc.Driver";
```

Reply:
Sorry for that mistake, Now the proper naming is done.

Reply:
Not updated